.circleci/config.yml

```diff
@@ -90,9 +90,19 @@ jobs:
             source emsdk_env.sh
             python scripts/test.py
 
+  build-docker-image:
+    machine: true
```

Comment:
Why this?

Reply:
This uses a docker machine executor which is required to build docker images.
https://circleci.com/docs/2.0/examples-intro/#linux-with-machine